src/PanelTraits/Update.php

```diff
@@ -38,26 +40,25 @@ public function update($id, $data)
     /**
      * Get all fields needed for the EDIT ENTRY form.
      *
-     * @param  [integer] The id of the entry that is being edited.
-     * @param int $id
+     * @param int $id The id of the entry that is being edited.
      *
-     * @return [array] The fields with attributes, fake attributes and values.
+     * @return array The fields with attributes, fake attributes and values
      */
     public function getUpdateFields($id)
     {
         $fields = $this->update_fields;
         $entry = $this->getEntry($id);
 
-        foreach ($fields as $k => $field) {
+        foreach ($fields as &$field) {
```

Comment:
This creates an additional 'value' entry inside the field array, if it wasn't already set. What would be the benefit of creating another array for that? Besides, the lines below this code does the same for the 'id' entry.